src/Tokenizer/Analyzer/FunctionsAnalyzer.php

```diff
@@ -127,6 +127,14 @@ public function isGlobalFunctionCall(Tokens $tokens, $index)
             return $functionUse->getShortName() === ltrim($functionUse->getFullName(), '\\');
         }
 
+        // check if is a part of an attribute: #[Foo(), Bar(), Baz()]
+        if (\defined('T_ATTRIBUTE')) {
+            $attributeStartIndex = $tokens->getPrevTokenOfKind($index, [[T_ATTRIBUTE]]);
```

Comment:
In case we are not in an attribute and start looking back, is there another way to see without going all the way back to the index zero?
For example, if we look back and find a `CT::T_ATTRIBUTE_CLOSE`, `T_OPEN_TAG` , `T_OPEN_TAG_WITH_ECHO`, `T_ATTRIBUTE`, `;` , `T_CLOSE_TAG`, `T_RETURN` can these be used inside an attribute?

Reply:
@SpacePossum yeah, let's go with more general approach: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5856